modules/RsIskraMeter/src/main.rs

```diff
@@ -1179,12 +1179,14 @@ mod tests {
             let ready_state = make_ready_state(mock);
 
             let _unused = ready_state.start_transaction(TransactionReq {
-                cable_id: 1,
-                client_id: String::new(),
                 evse_id: String::new(),
-                tariff_id: 1,
                 transaction_id: String::new(),
-                user_data: String::new(),
+                identification_status: generated::types::powermeter::OCMFUserIdentificationStatus::ASSIGNED,
```

Comment:
nice great job with fixing. There is a generated::types::powermeter include block at the top of the file. Lets add OCMFIdentificationType to it so we can write here just OCMFIdentificationType::ISO14443 and so on